### ISSUE TITLE
[Suggestion] Thaumostatic / Elytra Augment - Vis Battery Augment

### ISSUE THREAD

**Issue by LincT** [TheCodex6824/ThaumicAugmentation#362]
I didn't see a previous post for this subject, apologies if I missed it.
I thought since this augment already exists in game for the gauntlet, why not have them for the thaumostatic / elytra harness.

The Vis Battery augment could increase the vis capacity of either by 50% (bringing the thaumostatic harness at 300, the elytra at 75)

**Comment by TheCodex6824** [comment]
Yeah this would be a good addition. It would be nice to also have a second augment option to make this not just a direct upgrade over it, though.

**Comment by LincT** [comment]
Do you mean like the gyroscope allowing for stopping or at least negating all fall damage? Or if you're worried about being able to buff it too soon, maybe make it a stacked system so you have to have the impetus thruster first and craft it with the vis battery to make a vis buffered thruster, the lore being that the thruster also allows for addons increasing vis and impetus storage.

I thought about proposing a sort of rip-cord emergency stop that would be based on ender pearls and causing one to instantly teleport to the first non-air block below them, or instead of banners being able to craft firework effects like sparkles into flight, but not sure how hard that would be to implement.

Another idea might be for physical combat, an augment that gives a chance to stop damage to the player with the wings instead momentarily protecting the user, the materials of the void starting to become a little too intertwined in the caster's madness, but I wasn't sure if that was within theme. (I do appreciate how much this mod has stuck to a reasonable theme extension vs other mods adding in random op stuff and new essentia types because why not)

**Comment by TheCodex6824** [comment]
I'm not too worried about the balance of the augment itself (the vis storage probably wouldn't be massive anyway), I just like to make the augments involve having to make decisions / tradeoffs compared to each other instead of them just being direct upgrades.The augment helping in things other than flight (like combat) is an interesting idea though, I might have to borrow that one.

**Comment by LincT** [comment]
One other idea I had was an inlay that accepts a focus, and on impact it fires off the focus effect with increased cooldown.
Examples: 
- water focus - you're in the nether, and get hit by a blaze, it goes off and puts you out (assuming self > water or such)
- shield - you get hit once and your shield appears in front of you (maybe custom check where if already present do not fire again, or toggle it back off if you feel like mischief)
- air - you get hit and you push everyone away
- void - opens a void below you (if people want to do this, that's on them)
- exchange - swaps block below you with last imprinted
- heal - heals you
custom handling would need to be made for ranges such as items with touch/bolt would simply fire at the block below the wearer, projectile would fire as though the caster was looking straight up, spell bats would just appear above as well and then perform their function, clouds would be as normal where if touch/self/bolt centered on caster, projectiles happen where projectile lands.

recipe could involve an advanced brain or be based off similar to the auto caster?
